client/striplighttimer.py

```diff
@@ -0,0 +1,29 @@
+import requests
+import time
+import datetime
+headers={'accept': 'application/json'}
```

Comment:
```suggestion
HEADERS = {'accept': 'application/json'}
```
convention is to use UPPER_CASE_NAMES for constant values, and space before and after `=`